test/canary/resources/canary_config.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
This is placeholder config. Change when @jefchien and @adam-mateen figure out what metrics the canary should use.

Reply:
This is fine with me.
The purpose of this is to test downloading and installing from S3 yields the expected version.

Reply:
If its only running the agent, maybe create a SSM config and let's the agent pull so that we don't have to create a separate config/test for it.

Reply:
Talked with team, we decided to go with config.json instead of ssm since that will add an extra layer of obfuscation to the config. We would rather it be in the test dir code because it is easier to find.

Reply:
No I meant create a default template config in the terraform test, create it also in the ssm by using ssm resources and running the agent with /amazon-cloudwatch-agent-ctl --config ssm-config so we don't need to have a separate script for it.

Reply:
Canary test does not need to verify downloading configuration files from SSM.
That could be test earlier during the "integration test" phase.

If/when we release CCWA to SSM. In other words we push new SSM Documents that customers use to download and configure CCWA, then we would want a canary test that pulls them and verified they are the expected Documents.

Reply:
To clarify, the canary test are only running the CWA. Therefore, we don't need a **separate go file**  to run it, so an easiest way would be create a default config in terraform and using terraform to put it in the SSM configuration